front/app/containers/Admin/pagesAndMenu/containers/ContentBuilder/components/CraftComponents/Container/index.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+// components
+import { Box } from '@citizenlab/cl2-component-library';
+
+// craft
+import { UserComponent } from '@craftjs/core';
+import Events from '../Events';
+import Proposals from '../Proposals';
+import Projects from '../Projects';
+
+const Container: UserComponent = ({ children }) => {
+  return (
+    <Box id="e2e-single-column" minHeight="40px" w="100%">
+      {children}
+    </Box>
+  );
+};
+
+Container.craft = {
```

Comment:
The container is now duplicated to add this homepage-specific rules without affecting the rest of the code.